app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +69,121 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
\ No newline at end of file
+    @Test
+    @DisplayName("A cell that is dead can be set to alive")
+    void setAliveAt() {
+        Board board = new Board();
+
+        Boolean cellStatus = board.isCellAliveAt(1, 1);
+        assertFalse(cellStatus);
+
+        board.setAliveAt(1, 1);
+        cellStatus = board.isCellAliveAt(1, 1);
+
+        assertTrue(cellStatus);
+
+    }
+
+    @Nested
+    @DisplayName("Tests for getAliveNeighborsCountOf")
+    class GetAliveNeighborsCountOf {
+        public Board board;
+        @BeforeEach
+        void createBoard() {
+            board = new Board(3,3);
+        }
+        @Test
+        @DisplayName("A cell with 0 alive neighbor should return 0 alive neighbor")
+        void getAliveNeighborsCountOf() {
+            int numOfAliveNeighbors = board.getAliveNeighborsCountOf(1,1);
+            assertEquals(numOfAliveNeighbors, 0);
+        }
+
+        @Test
+        @DisplayName("A cell with 1 alive neighbor should return 1 alive neighbor")
+        void getAliveNeighborsCountOfTest1() {
+            board.setAliveAt(0,1);
+
+            int numOfAliveNeighbors = board.getAliveNeighborsCountOf(1, 1);
+
+            assertEquals(numOfAliveNeighbors, 1);
+        }
+
+        @Test
+        @DisplayName("A cell with 2 alive neighbors should return 2 alive neighbors")
+        void getAliveNeighborsCountOfTest2() {
+            board.setAliveAt(0,1);
+            board.setAliveAt(0,2);
+
+            int numOfAliveNeighbors = board.getAliveNeighborsCountOf(1,1);
+
+            assertEquals(numOfAliveNeighbors, 2);
+        }
+
+        @Test
+        @DisplayName("A cell with 8 alive neighbors should return 8 alive neighbors")
+        void getAliveNeighborsCountOfTest3() {
+            board.setAliveAt(0,0);
```

Comment:
Similar to above, if you find yourself doing this more than twice in the specs, I'd extract this to a helper method - `setEntireBoardAlive()` that iterates over any size board and sets every cell to be "alive"

Reply:
Added this helper as well